lib/chef/knife/bootstrap_windows_winrm.rb

```diff
@@ -37,6 +41,17 @@ class BootstrapWindowsWinrm < Bootstrap
       banner "knife bootstrap windows winrm FQDN (options)"
 
       def run
+        if (Chef::Config[:validation_key] && !File.exist?(File.expand_path(Chef::Config[:validation_key])))
+
+          if !negotiate_auth? && !(locate_config_value(:winrm_transport) == 'ssl')
+            ui.error("Validatorless bootstrap only supported with negotiate authentication protocol and ssl/plaintext transport")
+            exit 1
+          elsif !(Chef::Platform.windows?) && negotiate_auth?
+            ui.error("Negotiate protocol with plaintext transport is only supported when this tool is invoked from windows based system")
```

Comment:
Doesn't this duplicate the same error being raised in `Chef::Knife::WinrmCommandSharedFunctions.resolve_winrm_transport_options` (in winrm_knife_base.rb)?